gdnative-core/src/core_types/color.rs

```diff
@@ -47,6 +48,68 @@ impl Color {
             a: self.a + (weight * (other.a - self.a)),
         }
     }
+    #[inline]
+    pub fn blend(&self, other: &Color) -> Color {
+        Color::from_sys(unsafe { (get_api().godot_color_blend)(self.sys(), other.sys()) })
+    }
+
+    #[inline]
+    pub fn contrasted(&self) -> Color {
+        Color::from_sys(unsafe { (get_api().godot_color_contrasted)(self.sys()) })
+    }
+    #[inline]
+    pub fn darkened(&self, amount: f32) -> Color {
+        Color::from_sys(unsafe { (get_api().godot_color_darkened)(self.sys(), amount) })
+    }
+    #[inline]
+    pub fn from_hsv(h: f32, s: f32, v: f32, a: f32) -> Color {
```

Comment:
Should this maybe be named `hsv()`, consistent with the `rgb()` constructor -- or rename the latter?

Also, I would add overloads for HSV and HSVA.

Reply:
I'll go ahead and add those.

Reply:
I think it's better to rename `rgb()` here: `from_thing` is a common naming pattern for constructors in Rust.

Reply:
In my latest commit, I changed the name to `hsv()` and `hsva()` I can change it to follow either pattern as they both make sense to me.

Reply:
@toasteater @Bromeon after thinking this over, I was wondering if it would be possible to use the `from_*` naming pattern, doing so would likely break compatibility with anyone that is currently using color.

To avoid breaking compability, I could mark the rgb() and rgba() methods deprecated, if that's alright.

Reply:
@jacobsky That sounds reasonable! FWIW, we're working on a 0.10 release on `master`, so breaking compatibility is acceptable here, although this is useful to note since we might want to make a 0.9.4 release too. I think adding `from_*` but deprecating the old methods is good for now.

Reply:
@toasteater Thanks for the feedback. I'll get to work on that. I noticed that there were a few clippy lints that failed when I added the deprecated to rgb() and rgba(), so I want to take a little more time to ensure that I don't miss any of those changes.